COMPILING.md

```diff
@@ -46,6 +46,19 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
    make
    ```
 
+4. Linking against an IPASIR SAT solver
+
+   Get an IPASIR package and build picosat by default
+   ```
+   make -C ipasir-build
+   ```
+
+   Build CBMC with IPASIR and link against the ipasir solver library
+   Note: the LIBSOLVER variable could be pointed towards other solvers
+   ```
+   make -C src IPASIR=../../ipasir LIBSOLVER=$(pwd)/ipasir/libipasir.a
+   ```
```

Comment:
We've got the names for the libraries of the other solvers in the config; why not for IPASIR as well?

Then it would be enough to say
make -C src IPASIR=../../ipasir

Reply:
Because typically (except you always want to use picosat) you want to push people to select their most favourite IPASIR solver explicitly at compile time.

Reply:
ok!